python/modules/IcePy/ObjectAdapter.cpp

```diff
@@ -1807,7 +1807,7 @@ IcePy::wrapObjectAdapter(const Ice::ObjectAdapterPtr& adapter)
     {
         return 0;
     }
-    PyObject* wrapperType = lookupType("Ice.ObjectAdapterI");
+    PyObject* wrapperType = lookupType("Ice.ObjectAdapterI.ObjectAdapterI");
```

Comment:
I removed the import ObjectAdaterI from Ice/__init__.py as the file is not part of the public API, is just a wrapper used by IcePy.